README.Rmd

```diff
@@ -14,273 +23,92 @@ knitr::opts_chunk$set(
-
-- A process (expectation) module that models the expected counts by date of reference (`reference_date`)
-- A parametric reference reporting model which models the reporting delay distribution from the date of reference
-- A non-parametric reporting model which models differences in the reporting delay distribution by date of report (`report_date`), for example, day-of-the-week effects on the reporting delay.
-
-In the following sections, we specify simple models for each of these modules. The appropriateness of these specifications will vary depending on your context. See our vignettes for further details on model specification and examples of more complex models.
-
-#### Process model
-
-A commonly used process model in nowcasting is to model the expected counts by date of reference via a geometric random walk as this acts as a minimally informed smoothing prior and thus gives a lot of weight to the observed data. This is the default process model in `epinowcast`. Users may also specify this model for themselves using the enw_expectation() function.
-
-```{r}
-expectation_module <- enw_expectation(
-  ~ 0 + (1 | day), data = pobs
-)
-```
-
-Here, `day` refers to the number of days from the start of the data.
-
-As the underlying process model is an exponential growth rate model ($C_t = C_{t-1} \exp^{r_t}$), specifying a random effect (i.e. `(1 | day)`) on the growth rate is equivalent to a geometric random walk on expected counts by reference date.
-
-#### Reporting model by reference date
-
-Our baseline assumption for the reporting delay is that it is log-normally distributed, and static over time and strata. We can specify this model using the `enw_reference()` function,
-
-```{r}
-reference_module <- enw_reference(~1, distribution = "lognormal", data = pobs)
-```
+As you use the package, the documentation available via `?enw_` should be your first stop for troubleshooting. We also provide a range of other documentation, case studies, and spaces for the community to interact with each other. Below is a short list of current resources.
 
-Note that the default distribution is log-normal, hence the distribution argument could be omitted here. Alternatively we could model the reporting delay non-parametrically using a hazard model (see [our model description for details](https://package.epinowcast.org/articles/model)). The following is equivalent to a cox proportional hazards model with a single baseline hazard function.
+- [Package website](https://package.epinowcast.org/): This includes a function reference, model outline, and case studies using the package. The package site covers the release version, which can be installed from our Universe or from the latest GitHub release (see [installation instructions](#Installation)). Documentation for the development version (corresponding to the `main` branch on GitHub) [is also available](https://package.epinowcast.org/dev/).
```

Comment:
```suggestion
- [Package website](https://package.epinowcast.org/): This includes a function reference, model outline, and case studies using the package. The package site covers the release version, which can be installed from our Universe or from the latest GitHub release (see [installation instructions](#Installation)). Documentation for the development version (corresponding to the `main` branch on GitHub) [is also available](https://package.epinowcast.org/dev/).
- [Package Vignettes](https://package.epinowcast.org/articles): These provide tutorials, focused discussions of particular aspects, or demonstrate case studies. The [Getting Started with Epinowcast: Nowcasting](https://package.epinowcast.org/articles/getting-started-part-1) is a good place to start.
```

Reply:
this is good but case studies are in both